### PR TITLE
Updated badges to svg

### PR DESCRIPTION
**Pull request by amilajack**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

Updated badges to svg

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [x] I have read the [CONTRIBUTING](/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,10 +2,14 @@
 WebdriverIO
 ===========
 
-[![Build Status](https://travis-ci.org/webdriverio/webdriverio.png?branch=master)](https://travis-ci.org/webdriverio/webdriverio) [![Coverage Status](https://coveralls.io/repos/webdriverio/webdriverio/badge.png?branch=master&)](https://coveralls.io/r/webdriverio/webdriverio?branch=master) [![Code Climate](https://codeclimate.com/github/webdriverio/webdriverio/badges/gpa.svg)](https://codeclimate.com/github/webdriverio/webdriverio) [![Gitter](https://badges.gitter.im/Join Chat.svg)](https://gitter.im/webdriverio/webdriverio?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![Build Status](https://travis-ci.org/webdriverio/webdriverio.svg?branch=master)](https://travis-ci.org/webdriverio/webdriverio)
+[![NPM version](https://badge.fury.io/js/webdriverio.svg)](http://badge.fury.io/js/webdriverio)
+[![Dependency Status](https://img.shields.io/david/webdriverio/webdriverio.svg)](https://david-dm.org/webdriverio/webdriverio)
+[![npm](https://img.shields.io/npm/dm/thepiratebay.svg?maxAge=2592000)]()
```
Comment (christian-bromann) [inline README.md]: What is that?

**Comment by christian-bromann** [comment]
One comment

**Comment by amilajack** [comment]
It changes swaps the png request for a svg one.

**Comment by christian-bromann** [comment]
Right, but that one badge is not quite the right one 😆

**Comment by amilajack** [comment]
Sorry about that. I used another project of mine for boilerplate and forgot to change that.

**Comment by christian-bromann** [comment]
Thanks, now I have to update the dependencies to get at least one more green badge :-P

**Pull request merged**